pontoon/localizations/views.py

```diff
@@ -97,6 +99,10 @@ def ajax_resources(request, code, slug):
     for part in parts:
         part['resource__priority'] = resource_priority_map.get(part['title'], None)
 
+        deadline = part['resource__deadline']
+        if deadline:
+            part['resource__deadline'] = datetime.strptime(deadline, '%Y-%m-%d')
```

Comment:
I would do this in the template instead of the view, and keep the full `datetime` object here. And to turn stuff into that format, you can use `my_datetime.date().isoformat()`. See https://docs.python.org/2.7/library/datetime.html#datetime.date.isoformat